routes/web.php

```diff
@@ -41,3 +42,4 @@
 Route::post('/product/create', [ProductController::class, 'store'])->name('storeProduct');
 
 
+Route::get('/add/cart/{cart}', [CartController::class, 'show'])->name('cart');
```

Comment:
Is the route endpoint correct Like this? I think it should be something like `cart/add/{cart} `

Reply:
corrected